app/controllers/api/auth/accounts_controller.rb

```diff
@@ -9,15 +9,22 @@ class Api::Auth::AccountsController < Api::AccountsController
 
   filter_resources_by :user_id
 
+  before_filter :ensure_write_scope, except: [:index, :show]
+
   def create_resource
     return super unless params['user_id']
+
     account_user = User.find(params['user_id'])
     account_user.create_individual_account
     account_user.individual_account
   end
 
   private
 
+  def ensure_write_scope
+    user_not_authorized unless prx_auth_token.scopes&.include? 'account:write'
```

Comment:
This is just a placeholder for now, scope currently doesn't exist. I'm using this colon separated deal for permissions on an entity which I've seen e.g. in the [Slack API](https://api.slack.com/docs/oauth-scopes).